media-libs/gst_plugins_base/gst_plugins_base-1.26.3.recipe

```diff
@@ -0,0 +1,153 @@
+SUMMARY="GStreamer Base Plug-ins"
```

Comment:
This won't work the same as with the others, main package "gstreamer" is still at 1.26.2, so this hasn't been build against that, and will faill also, my guess is that you didn't check building it?